src/s3.js

```diff
@@ -3,7 +3,21 @@ var denodeify = require('denodeify')
 
 module.exports = async function(cfg) {
 
+    // var credentials = new AWS.SharedIniFileCredentials(cfg.aws_profile ? {profile: cfg.aws_profile} : {});
+    // AWS.config.credentials = credentials;
 
+    AWS.CredentialProviderChain.defaultProviders = [
+        function () { return new AWS.EC2MetadataCredentials(); },
+        function () { return new AWS.EnvironmentCredentials('AWS'); },
```

Comment:
Just for my understanding, are we likely to rely on env vars within apps?

Reply:
That makes sense. I've made a few changes after testing these changes locally in gudocs ([0a2e94b](https://github.com/guardian/koa-gu/pull/4/commits/0a2e94b3dd7eb5a9f4e2522eb0cc059fb190c5c1), [43398f4](https://github.com/guardian/koa-gu/pull/4/commits/43398f46860bf83726cec7c286d3b9d7fe1e22df)), and that's now working locally.